src/Particle/FastParticleOperators.h

```diff
@@ -199,149 +184,150 @@ struct ConvertPosUnit<ParticleAttrib<TinyVector<T,3> >,Tensor<T,3>, 3, false>
+      pout[i][1] = pin[i][0] * x01 + pin[i][1] * x11;
     }
   }
 
-  inline static void
-  apply(const Transformer_t& X, const Array_t& pin,  Array_t& pout, int first, int last)
+  inline static void apply(const Transformer_t& X, const Array_t& pin, Array_t& pout, int first, int last)
   {
-    register T x00=X[0],x01=X[1],x10=X[2],x11=X[3];
+    register T x00 = X[0], x01 = X[1], x10 = X[2], x11 = X[3];
 #pragma ivdep
-    for(int i=first; i<last; i++)
+    for (int i = first; i < last; i++)
     {
-      pout[i][0]=pin[i][0]*x00+pin[i][1]*x01;
-      pout[i][1]=pin[i][0]*x10+pin[i][1]*x11;
+      pout[i][0] = pin[i][0] * x00 + pin[i][1] * x01;
+      pout[i][1] = pin[i][0] * x10 + pin[i][1] * x11;
     }
   }
 
-  inline static void
-  apply(Array_t& pinout, const Transformer_t& X,int first, int last)
+  inline static void apply(Array_t& pinout, const Transformer_t& X, int first, int last)
   {
-    register T x00=X[0],x01=X[1],x10=X[2],x11=X[3];
+    register T x00 = X[0], x01 = X[1], x10 = X[2], x11 = X[3];
 #pragma ivdep
-    for(int i=first; i<last; i++)
+    for (int i = first; i < last; i++)
     {
-      T _x(pinout[i][0]),_y(pinout[i][1]);
-      pinout[i][0]=_x*x00+_y*x10;
-      pinout[i][1]=_x*x01+_y*x11;
+      T _x(pinout[i][0]), _y(pinout[i][1]);
+      pinout[i][0] = _x * x00 + _y * x10;
+      pinout[i][1] = _x * x01 + _y * x11;
     }
   }
 
-  inline static void
-  apply(const Transformer_t& X, Array_t& pinout, int first, int last)
+  inline static void apply(const Transformer_t& X, Array_t& pinout, int first, int last)
   {
-    register T x00=X[0],x01=X[1],x10=X[2],x11=X[3];
+    register T x00 = X[0], x01 = X[1], x10 = X[2], x11 = X[3];
 #pragma ivdep
-    for(int i=first; i<last; i++)
+    for (int i = first; i < last; i++)
     {
-      T _x(pinout[i][0]),_y(pinout[i][1]);
-      pinout[i][0]=_x*x00+_y*x01;
-      pinout[i][1]=_x*x10+_y*x11;
+      T _x(pinout[i][0]), _y(pinout[i][1]);
+      pinout[i][0] = _x * x00 + _y * x01;
+      pinout[i][1] = _x * x10 + _y * x11;
     }
   }
 };
 
-#define SUPERCELL_BOUNDARY_LIMITS(T)               \
-  const T epsilon = -std::numeric_limits<T>::epsilon(); \
+#define SUPERCELL_BOUNDARY_LIMITS(T)                     \
+  const T epsilon  = -std::numeric_limits<T>::epsilon(); \
   const T plus_one = 1.0
 
 
-#define THREE_DIM_BOUNDARY_BLOCK(X,Y,Z,EPS,PLUSONE) \
-    if(X<EPS)           X+=PLUSONE;                 \
-    else if(X>=PLUSONE) X-=PLUSONE;                 \
-    if(Y<EPS)           Y+=PLUSONE;                 \
-    else if(Y>=PLUSONE) Y-=PLUSONE;                 \
-    if(Z<EPS)           Z+=PLUSONE;                 \
-    else if(Z>=PLUSONE) Z-=PLUSONE
-
-#define TWO_DIM_BOUNDARY_BLOCK(X,Y,EPS,PLUSONE) \
-    if(X<EPS)           X+=PLUSONE;                 \
-    else if(X>=PLUSONE) X-=PLUSONE;                 \
-    if(Y<EPS)           Y+=PLUSONE;                 \
-    else if(Y>=PLUSONE) Y-=PLUSONE
+#define THREE_DIM_BOUNDARY_BLOCK(X, Y, Z, EPS, PLUSONE) \
```

Comment:
I prefer the original formatting on this macro - putting the conditional and the expression on the same line makes it compact, and easier to see regularity of +=/-= operators
Any other opinions?  Is worth it to put the format exception markers around these macros?

Reply:
I think its just time to remove that macro.  However its heaviest presence is in QMCWaveFunctions. I'll look at adding them to this PR.